src/System.Data.SqlClient/src/System/Data/SqlClient/SqlCommand.cs

```diff
@@ -3189,42 +3222,42 @@ private void GetRPCObject(int paramCount, ref _SqlRPC rpc)
                 rpc = _rpcArrayOf1[0];
             }
 
-            rpc.ProcID = 0;
-            rpc.rpcName = null;
+            //rpc.ProcID = 0;
+            //rpc.rpcName = null;
```

Comment:
Why are these commented out?  If they're not necessary, should the lines be deleted.  If they're commented out to highlight something, I think that deserves a comment saying why.

Reply:
removed